components/BackButton/index.test.jsx

```diff
@@ -0,0 +1,28 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { useRouter } from 'next/router';
+
+import BackButton, { backButtonTestIds } from '.';
+
+describe('BackButton', () => {
+  it('renders a back button', () => {
+    render(<BackButton />);
+
+    const backButton = screen.getByTestId(backButtonTestIds.button);
+
+    expect(backButton).toBeVisible();
+  });
+
+  it('goes to the previous page upon click', () => {
+    const mockRouter = { back: jest.fn() };
+    useRouter.mockReturnValue(mockRouter);
+
+    render(<BackButton />);
+
+    const backButton = screen.getByTestId(backButtonTestIds.button);
+    fireEvent.click(backButton);
+
+    expect(backButton).toBeVisible();
```

Comment:
Could you point me out why do we need to expect the back button to be visible after clicking it? 🤔

Reply:
dropped it on d129258d877c1e23ba31313cc0390ad9627eb192, we actually don't need it 🤭